Docker image does not contain the requirements to upgrade to v2.4

Instructions on [How to Upgrade](https://github.com/wallabag/wallabag/releases/tag/2.4.0) do not work within the Docker image, since `make` and `bash` are absent from the image.

### Environment
 
* **Version**: 2.4.0-2-g749587f (`git describe --tags`)
* **Installation**: Docker
* **PHP version**: 7.3.25
* **OS**: Alpine Linux v3.12 (Docker image)
* **Database**: SQLite
* **Parameters**:

### What steps will reproduce the bug?

Get a shell in the Docker container:
```sh
docker exec -ti --user nobody wallabag-container sh
~ $ cd /var/www/wallabag
/var/www/wallabag $ make update
sh: make: not found

(... install make from another terminal as root)

/var/www/wallabag $ make update
make: bash: No such file or directory
make: *** [GNUmakefile:24: update] Error 127
```

### Suggestion to solve the issue

Add `apk add make bash` in the Dockerfile.

--- a/Dockerfile
+++ b/Dockerfile
@@ -51,6 +51,8 @@ RUN set -ex \
       s6 \
       tar \
       tzdata \
+      make \
+      bash \
  && rm -rf /var/cache/apk/* \
  && ln -sf /dev/stdout /var/log/nginx/access.log \
  && ln -sf /dev/stderr /var/log/nginx/error.log \

